### PR TITLE
🌐 Adds Italian translation

### PR DESCRIPTION
**Pull request by simonecorsi**
Hey 👋  Nice project! 🚀 

I was reading the documentation for some ideas and thought that it could be useful to have an Italian translation so more people can learn something from it, so here it comes!

### CODE REVIEW

File: README-it.md [hunk 1]
```diff
@@ -0,0 +1,325 @@
...
+- [پارسی](./README-fa.md)
+- [Francese](./README-fr.md)
+- [Italiano](./README-it.md)
+- [日本語](./README-jp.md)
+- [Portoghese](./README-pt.md)
+- [Russo](./README-ru.md)
+- [සිංහල](./README-si.md)
+- [Turco](./README-tr.md)
+- [简体中文](./README-zh-cn.md)
+- [繁體中文](./README-zh-tw.md)
```
Comment (nlepage) [inline README-it.md:31]: Usually we leave each language name written in its own language, so `Inglese` -> `English` etc.

File: README-it.md [hunk 2]
```diff
@@ -0,0 +1,325 @@
...
+  <a href="https://twitter.com/intent/tweet?text=How%20to%20promote%20your%20open-source%20projects%20@ZenikaOSS&url=https://github.com/zenika-open-source/open-source-promotion-cheat-sheet&hashtags=OpenSource,CheatSheet">
+    <img alt="tweet" src="https://img.shields.io/twitter/url/https/twitter?label=Share%20on%20twitter&style=social" target="_blank" />
+  </a>
+</p>
+
+<h1 align="center">Promuovere progetti open source</h1>
+
+> In questi appunti sono riassunti alcuni passaggi importanti da seguire per poter promuovere efficacemente il vostro progetto open source. Potete cliccare un elemento per espandere l'argomento ed avere più informazioni.
+
+List of available languages:
```
Comment (nlepage) [inline README-it.md:15]: I think you forgot this line.

**Review by nlepage — COMMENTED** [review]
Hi @simonecorsi 
Thanks a lot for your contribution!
We'll try to review it ASAP.
Meanwhile I left you some minor comments.
Cheers!

File: README-it.md (hunk 1 above)
Comment (simonecorsi) [reply README-it.md]: Looks good! I didn't think about that! 
Pushed the fixed list.

File: README-it.md (hunk 2 above)
Comment (simonecorsi) [reply README-it.md]: Translated!

**Comment by simonecorsi** [comment]
Let me know if you want me to squash the commits after your reviews :+1:

**Review by atogue — APPROVED** [review]
Good translation, suggestions fluent and easy to understand!

**Review by nlepage — APPROVED** [review]

**Comment by nlepage** [comment]
Thanks @simonecorsi and @atogue !

**Pull request merged**